system/ee/EllisLab/Tests/ExpressionEngine/Service/Validation/ValidatorTest.php

```diff
@@ -50,6 +50,8 @@ public function testRequired()
 
 	public function testChaining()
 	{
+		$this->markTestSkipped('Skipping because of ee_mb_strlen not being found');
```

Comment:
I don't understand why this is here? @intoeetive  is the function just not available here for some reason?

Reply:
I wasn't able to figure out how to perperly load the legacy helper for unit tests and so was skipping - but now I know way to do this and have separate PR https://github.com/ExpressionEngine/ExpressionEngine/pull/529